### PR TITLE
Updated screenshots agent dashboard: inventory, stats, and configuration

### LINKED ISSUE #7941: Update Wazuh dashboard agents screenshots and remove VirusTotal references
# Description

In Wazuh 4.10.0 the agent views have many changes that we need to reflect in Wazuh documentation. We also integrated **VirusTotal** module into **Malware detection**, therefore we have to remove the references to VirusTotal and update any Malware Detection dashboard screenshot.

> [!note]
> For each task create a pull request to make it easier to review and integrate.

### Tasks 
- [x] #7944
- [x] #7942
- [x] #7946
- [x] #7965

### PR DESCRIPTION
**Pull request by guidomodarelli**
## Description
Update screenshots of the agent dashboard and its sub-sections: Inventory / Stats / Configuration.

### Issue resolved
#7941

### Docs building
- [ ] Compiles without warnings.
### Code formatting and web optimization
- [ ] Uses three spaces indentation.
- [ ] Adds or updates meta descriptions accordingly.
- [ ] Updates the `redirects.js` script if necessary (check [this guide](https://github.com/wazuh/wazuh-documentation/blob/master/NEW_RELEASE.md)).
### Writing style
- [ ] Uses present tense, active voice, and semi-formal registry.
- [ ] Uses short, simple sentences.
- [ ] Uses **bold** for user interface elements, _italics_ for key terms or emphasis, and `code` font for Bash commands, file names, REST paths, and code.

### CODE REVIEW

File: source/images/wazuh-dashboard/queries/port-less-than-value.png
Comment (Desvelao) [inline source/images/wazuh-dashboard/queries/port-less-than-value.png:1]: **nitpick**: The screenshot should take into account the surrounding text (or vice versa).

File: source/images/wazuh-dashboard/queries/port-ranges.png
Comment (Desvelao) [inline source/images/wazuh-dashboard/queries/port-ranges.png:1]: **nitpick**: The screenshot should take into account the surrounding text (or vice versa).

File: source/images/wazuh-dashboard/queries/protocol-not-ipv6.png
Comment (Desvelao) [inline source/images/wazuh-dashboard/queries/protocol-not-ipv6.png:1]: **nitpick**: there are 2 network interfaces with the same name and different IP addresses. Could you remove one of them of mocked data and take the screenshot again?

**Comment by Machi3mfl** [comment]
# Test

## Queries > Greater than - Less than

Screenshot 2024-11-05 at 11 12 21 AM

## Queries > Value

Screenshot 2024-11-05 at 11 23 01 AM

## Queries > Not ipv6

Screenshot 2024-11-05 at 11 23 58 AM

**Review by Machi3mfl — APPROVED** [review]

**Pull request merged**